judge/migrations/0150_submission_is_archived.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 4.2.17 on 2025-03-02 10:16
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('judge', '0149_add_organization_private_problems_permission'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='submission',
+            options={
+                'permissions': (
+                    ('abort_any_submission', 'Abort any submission'),
+                    ('rejudge_submission', 'Rejudge the submission'),
+                    ('rejudge_submission_lot', 'Rejudge a lot of submissions'),
+                    ('spam_submission', 'Submit without limit'),
+                    ('view_all_submission', 'View all submission'),
+                    ('resubmit_other', "Resubmit others' submission"),
+                    ('lock_submission', 'Change lock status of submission'),
+                    ('archive_submission', 'Archive any submission'),
+                ),
+                'verbose_name': 'submission',
+                'verbose_name_plural': 'submissions',
+            },
+        ),
```

Comment:
Err, what does this part of the migration do?

Reply:
this is needed so that the permission `archive_submission` appears under `/admin/auth/user/<id>/change/ > Available user permissions`